components/constants/BUILD.gn

```diff
@@ -0,0 +1,41 @@
+import("//brave/build/config.gni")
+import("//build/buildflag_header.gni")
+import("//build/util/branding.gni")
+import("//mojo/public/tools/bindings/mojom.gni")
+
+declare_args() {
+  brave_services_key = ""
+}
+
+if (is_official_build) {
+  assert(brave_services_key != "")
+}
+
+buildflag_header("brave_services_key") {
+  header = "brave_services_key.h"
+  flags = [ "BRAVE_SERVICES_KEY=\"$brave_services_key\"" ]
+}
+
+source_set("constants") {
+  sources = [
```

Comment:
Not sure that we should limit visibility of the source set (via 'visibility = [...]`).
It's safe to include anywhere, because we don't have any sophisticated deps.